### PR TITLE
Added Scraping and Parsing functionality for ComputerJobs

### PR DESCRIPTION
**Pull request by turrence**
- Added scraping and parsing functionality to ComputerJobs
- Added new column "Source" to csv creation
- Refactored to account for more than 1 scraper

### CODE REVIEW

File: src/scrape/scrape_linkedin.py [hunk 1]
```diff
@@ -45,8 +45,8 @@ def start_requests(self):
 
     def parse_job_list(self, response):
         job_list = [
-            urljoin(url, urlparse(url).path)
-            for url in response.xpath("/html/body/li/div/a").xpath("@href").getall()
+            urljoin(url, urlparse(url).path) 
+                for url in response.xpath("/html/body/li/div/a").xpath("@href").getall()
```
Comment (jaasonw) [inline src/scrape/scrape_linkedin.py:49]: is this indent valid python?

File: src/transform/parse_computerjobs.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+)
+
+def convert_time_to_isoformat(date_string):
+    date_format = "%m/%d/%Y %I:%M:%S %p"
+    date = datetime.strptime(date_string, date_format)
+    date = date.astimezone(timezone("US/Pacific"))
+    return date.isoformat()
+
+
+def parse_computerjobs(metadata):
```
Comment (jaasonw) [inline src/transform/parse_computerjobs.py]: is this significantly different enough such that it cant be deduped into 1 parse function?

**Review by jaasonw — CHANGES_REQUESTED** [review]

File: src/scrape/scrape_linkedin.py (hunk 1 above)
Comment (turrence) [reply src/scrape/scrape_linkedin.py:49]: IT RAN ON MY MACHINE.

File: src/transform/parse_computerjobs.py (hunk 2 above)
Comment (turrence) [reply src/transform/parse_computerjobs.py]: i guess not. it handles the metadata differently tho

File: src/scrape/scrape_linkedin.py (hunk 1 above)
Comment (jaasonw) [reply src/scrape/scrape_linkedin.py:49]: works 4 me xd

**Comment by jaasonw** [comment]
LGTM!

**Pull request merged**